agents/ipmilan/fence_ipmilan.py

```diff
@@ -206,6 +208,7 @@ def main():
 	docs["symlink"] = [("fence_ilo3", "Fence agent for HP iLO3"),
 		("fence_ilo4", "Fence agent for HP iLO4"),
 		("fence_ilo5", "Fence agent for HP iLO5"),
+                ("fence_ipmitool", "Fence agent for IPMIv2 lanplus"),
```

Comment:
I think fence_ipmilanplus would be a more fitting name.

Also, ensure indentation follows the rest of the agent (tabs in this case).

Reply:
There is cobbler making use of fence_ipmitool (maybe others as well?).
Not sure where it exactly came from. It at least was not a link, but a whole bunch of code. I tried to get around inside cobbler by this. But as lanplus can come via stdin or param, this did not really work:
#!/bin/bash -x

# fence_ipmitool wrapper script to call fence_ipmilan with -P parameter
# fence_ipmilan will then use IPMIv2.0 -I lanplus interface which should
# be used by all modern IPMI driven BMCs out there

# An issue is open for fence-agents project to use IPMIv2.0 by default:
# https://github.com/ClusterLabs/fence-agents/issues/314

# Clarification:
# fence_ipmilan "$@" --lanplus
# does not work. cobbler passes options via stdin and fence_ipmilan
# does either recognise opions from stdin or via commandline.
# Therefore below line adds lanplus=1\n to incomming stdin (cobbler passed fence
# options)

cat <(echo "lanplus=1") - | fence_ipmilan "$@"


I'd appreciate if either:
- the new fence is called fence_ipmitool
or
- the two links are made up fence_ipmitool and fence_ipmilanplus

Otherwise I have to think about a solution in our package only, either cobbler or fence-agents which again could lead to such unforeseeable incompatibilities in some years...

Thanks.

PS: Sry about the tabs, forgot to double check...

Reply:
Is this on Suse? I'll add a commit to add the fence_ipmilanplus in general, and only add fence_ipmitool to builds on distros where it's expected.

Reply:
Argh..., indeed. I just grepped through the cobbler project. No ipmitool fence agent. I thought this is distro unspecific.
No need for further ipmitool dirtyness then in the upstream project. I can handle this on our side.
I'll update to ipmilanplus.
Thanks for the very quick respone!

Reply:
Perfect. You're welcome.